crates/strategies/opensea-sudo-arb/contracts/README.md

```diff
@@ -0,0 +1,262 @@
+| selector                                 | 4             |
+| bop_ptr                                  | 30            |
+| payment_value                            | 30            |
+| sudo_pool                                | 30            |
+| bop_consideration_token                  | 30            |
+| bop_consideration_identifier             | 30            |
+| bop_consideration_amount                 | 30            |
+| bop_offerer                              | 30            |
+| bop_zone                                 | 30            |
+| bop_offer_token                          | 30            |
+| bop_offer_identifier                     | 30            |
+| bop_offer_amount                         | 30            |
+| bop_start_time                           | 30            |
+| bop_end_time                             | 30            |
+| bop_zone_hash                            | 30            |
+| bop_salt                                 | 30            |
+| bop_offerer_conduit_key                  | 30            |
+| bop_fulfuller_conduit_key                | 30            |
+| bop_total_original_additional_recipients | 30            |
+| bop_additional_recipients_ptr            | 30            |
+| bop_signatures_ptr                       | 30            |
+| bop_additional_recipients_len            | 30            |
+| bop_additional_recipients                | 60 * len      |
+| signature_len                            | 30            |
+| signature                                | signature_len |
+
+### `withdraw()`
+
+Calldata layout:
+
+```
+selector
+```
+
+| name     | size (bytes) |
+| -------- | ------------ |
+| selector | 4            |
+
+### `transferOwnership(address)`
+
+Calldata layout:
+
+```
+selector . new_owner
+```
+
+| name      | size (bytes) |
+| --------- | ------------ |
+| selector  | 4            |
+| new_owner | 32           |
+
+### `owner() view returns (address)`
+
+Calldata layout:
+
+```
+selector
+```
+
+| name     | size (bytes) |
+| -------- | ------------ |
+| selector | 4            |
+
+## Solidity x Yul Reference Implementation
+
+```solidity
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.13;
+
+import {Seaport} from "../src/protocols/Seaport/contracts/Seaport.sol";
+import {LSSVMPairETH} from "../src/protocols/LSSVMPairFactory/contracts/LSSVMPairETH.sol";
+import {BasicOrderParameters} from "../src/protocols/Seaport/contracts/lib/ConsiderationStructs.sol";
+import {IERC721} from "../src/protocols/LSSVMPairFactory/contracts/imports/IERC721.sol";
+import {Owned} from "solmate/auth/Owned.sol";
+
+contract SudoOpenseaArbFast is Owned {
+
+    error NoProfit();
+
+    constructor() Owned(msg.sender) {}
+
+    Seaport constant seaport = Seaport(0x00000000000001ad428e4906aE43D8F9852d0dD6);
+
+    /// @custom:unsafe-as-fuck
+    function executeArb(BasicOrderParameters calldata, uint256 paymentValue, address payable sudo_pool) public {
```

Comment:
could we please add comments explaining what some of the constants are in the reference implementation? would make it a bit easier to follow along.

Reply:
added constant names to clarify each variables' role. lmk if there should be more context